valohai_yaml/pipelines/conversion.py

```diff
@@ -55,22 +55,24 @@ def convert_deployment_node(self, node: DeploymentNode) -> ConvertedObject:
     def convert_executionlike_node(self, node: Union[ExecutionNode, TaskNode]) -> ConvertedObject:
         node_data = node.serialize()
         step_name = node_data.pop("step")
-        override = node_data.pop("override", {})
+
         step = self.config.get_step_by(name=step_name)
         if not step:  # pragma: no cover
             raise ValueError(f"Step {step_name} not found in {self.config}")
         step_data = step.serialize()
-        step_data.update(override)
+        override = node_data.pop("override", None)
```

Comment:
Should this use `Override.parse()`..?

Reply:
it would be wrong format, cos step_data is serialised. node_data has a serialised override so inputs and parameters can be easily merged

Reply:
Yes, but you could parse the Override, merge things using the parsed objects (as opposed to dicts) and then serialize back?